test/topology_custom/test_zero_token_nodes_multidc.py

```diff
@@ -82,7 +84,9 @@ async def test_zero_token_nodes_multidc_basic(manager: ManagerClient, zero_token
                 continue  # EACH_QUORUM is supported only for writes
 
             select_query = SimpleStatement(f'SELECT * FROM {ks_names[rf]}.tbl', consistency_level=cl)
-            dc1_result = list((await dc1_cql.run_async(select_query))[0])
-            dc2_result = list((await dc2_cql.run_async(select_query))[0])
-            assert dc1_result == [1, 1]
-            assert dc2_result == [1, 1]
+            dc1_result_set = await dc1_cql.run_async(select_query)
+            dc2_result_set = await dc2_cql.run_async(select_query)
+            assert dc1_result_set
+            assert list(dc1_result_set[0]) == [1, 1]
```

Comment:
Patch comment says
> Replace IndexError-prone direct indexing with ResultSet.one()
but it looks like indexing is still here

Reply:
yeah, the commit message is out-dated. seems `run_async()` does not return a ResultSet -- it returns a list. the latest revision was updated, but the commit message was not. lemme update the commit message.